### PR TITLE
Add two Smalltalk books in pt-BR

### PR DESCRIPTION
**Pull request by tecnotercio**
## What does this PR do?
Add resources.

### Description
Two books on Smalltalk in Brazilian Portuguese.

### Why is this valuable (or not)?
There are no Smalltalk books in Brazilian Portuguese in the current list.

### How do we know it's really free?
The books were made available by the authors (one through ResearchGate and the other by the university where the author works).

### For book lists, is it a book? For course lists, is it a course? etc.
They are two different books, unrelated to each other.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -395,6 +396,13 @@
 * [Fundamentos de Sistemas Embarcados](https://www.youtube.com/playlist?list=PLqvo6YdcIqXXGY1dLbf_xA-JLMBumTyzG) - Renato Sampaio
 
 
+### Smalltalk
+
+* [Conhecendo o SmallTalk](https://www.researchgate.net/publication/262882317_Conhecendo_o_Smalltalk_-_Todos_os_Detalhes_da_Melhor_Linguagem_de_Programacao_Orientada_a_Objetos) - Daniel Duarte Abdala e
+Aldo von Wangenheim (PDF)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:402]: need to have this all on one line

Comment (eshellman) [reply courses/free-courses-pt_BR.md:402]: and separate authors with a comma

**Review by eshellman — CHANGES_REQUESTED** [review]
good resources

File: courses/free-courses-pt_BR.md (hunk 1 above)
Comment (tecnotercio) [reply courses/free-courses-pt_BR.md]: @eshellman, I made the changes as requested.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**